README.md

```diff
@@ -290,6 +291,26 @@ Or for all roots:
 MoabStorageRoot.find_each { |msr| Audit::MoabToCatalog.seed_catalog_for_dir(msr.storage_location) }
 ```
 
+## <a name="migrate_moab_manually"> Update the catalog when moving a Moab to a different storage root
+
+Sometimes it's necessary to move Moabs from one storage root to another, either in bulk as part of a storage hardware migration, or manually for a small number, as when an existing Moab might get too much additional content for the storage root it's currently on.
+
+There are rake tasks and documentation to suppor the bulk migration scenario.  See the [migration issue template](.github/ISSUE_TEMPLATE/storage-migration-checklist.md).
```

Comment:
suppor is typo.

Reply:
suppor catch , thanks! 😄

Reply:
just pushed a fix, should be the only change:
```
$ git diff origin/move-moab-migration-method-avoid-alliteration-assiduously 
diff --git a/README.md b/README.md
index 13875e00..941a3623 100644
--- a/README.md
+++ b/README.md
@@ -295,7 +295,7 @@ MoabStorageRoot.find_each { |msr| Audit::MoabToCatalog.seed_catalog_for_dir(msr.
 
 Sometimes it's necessary to move Moabs from one storage root to another, either in bulk as part of a storage hardware migration, or manually for a small number, as when an existing Moab might get too much additional content for the storage root it's currently on.
 
-There are rake tasks and documentation to suppor the bulk migration scenario.  See the [migration issue template](.github/ISSUE_TEMPLATE/storage-migration-checklist.md).
+There are rake tasks and documentation to support the bulk migration scenario.  See the [migration issue template](.github/ISSUE_TEMPLATE/storage-migration-checklist.md).
 
 When the need for moving a single Moab arises, the repository manager or a developer should:
 1. [on the file system] Move the Moab to a storage root with enough space
```